src/components/ShareTrajectoryModal/EmbedSnippetPanel.tsx

```diff
@@ -59,75 +62,105 @@ const EmbedSnippetPanel = ({ startTime }: EmbedSnippetPanelProps) => {
+                    className={classNames(styles.embedSettings, {
+                        [styles.open]: showEmbedSettingsPanel,
+                    })}
                 >
-                    Show settings {DownArrow}
-                </button>
-            </div>
-            <div
-                className={classNames(styles.embedSettings, {
-                    [styles.open]: showEmbedSettingsPanel,
-                })}
-            >
-                {showEmbedSettingsPanel && (
-                    <VerticalFlexbox gap={4}>
-                        <p className={styles.accentText}> Size </p>
-                        <VerticalFlexbox gap={24}>
-                            <div className={styles.proportionSettings}>
-                                {" "}
-                                <Input
-                                    defaultValue={height}
-                                    className={styles.numberInputs}
-                                    onChange={(e) => {
-                                        handleChangeHeight(e.target.value);
-                                    }}
-                                />
-                                <span> x </span>
-                                <Input
-                                    value={width}
-                                    className={styles.numberInputs}
-                                    onChange={(e) => {
-                                        handleChangeWidth(e.target.value);
-                                    }}
-                                />
-                                <label> pixels </label>
-                                <label className={styles.constrainProportions}>
-                                    <Checkbox
-                                        onChange={() => {
-                                            setConstrainProportions(
-                                                !constrainProportions
-                                            );
+                    {showEmbedSettingsPanel && (
+                        <VerticalFlexbox gap={4}>
+                            <p className={styles.accentText}> Size </p>
+                            <VerticalFlexbox gap={24}>
+                                <HorizontalFlexbox gap={8} alignItems="center">
+                                    <Input
+                                        defaultValue={height}
+                                        className={styles.numberInputs}
+                                        onChange={(e) => {
+                                            handleChangeHeight(e.target.value);
                                         }}
-                                    ></Checkbox>
-                                    Constrain proportions
-                                </label>
-                            </div>
+                                    />
+                                    <span> x </span>
+                                    <Input
```

Comment:
this is my fault, but these two should be switched: it should be w x h, not h x w. especially since they're not labeled